emulator/src/Emulator.cpp

```diff
@@ -9,11 +9,14 @@
 #include "../include/Display.h"
 #include "../include/Emulator.hpp"
 
+#include "osw_config.h"
+#include "osw_config_keys.h"
+
 OswEmulator* OswEmulator::instance = nullptr;
 
 OswEmulator::OswEmulator() {
     // Init the SDL window and renderer
-    this->mainWindow = SDL_CreateWindow("OSW-OS Emulator", SDL_WINDOWPOS_CENTERED, SDL_WINDOWPOS_CENTERED, DISP_W + this->guiPadding + this->guiWidth + this->guiPadding, DISP_H, SDL_WINDOW_RESIZABLE | SDL_WINDOW_ALLOW_HIGHDPI);
```

Comment:
Please revert this. This was intended

Reply:
It draws an overlay, which shows a display border... We should do that in SDL directly - as this is a job of the emulator.